tests/test_easycheck.py

```diff
@@ -137,6 +140,36 @@ def test_check_if_not_negative_warnings():
         assert issubclass(w[-1].category, Warning)
         assert "This is a testing warning" in str(w[-1].message)
 
+def test_check_if_in_limits():

```

Comment:
Please add 
* the same number of tests for floats, including very small and very big ones, with short and long intervals.
* 1-2 tests with a different exception